x-pack/legacy/plugins/canvas/types/canvas.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface CanvasElement {
+  id: string;
+  position: {
```

Comment:
There is a `Position` type in the `elements.ts` types that looks very similar to this.  Maybe those can be consolidated.